test_unit.py

```diff
@@ -1209,6 +1209,36 @@ def test_HTLC(self):
         self.assertEqual(p, base64.b64decode(golden))
         self.assertEqual(s.get_address(), golden_addr)
 
+    def test_periodic_fee(self):
```

Comment:
Does this test pass if it uses the test vector at https://algorand.atlassian.net/wiki/spaces/FDE/pages/445513729/SDK+Templates+integration#Testing-Vectors.3 ?

Reply:
Changed to use that test vector below.